setup.cfg

```diff
@@ -26,6 +26,9 @@ install_requires =
     numpy==1.16.0; python_version=='3.6' and platform_system=='AIX'
     numpy==1.16.0; python_version=='3.7' and platform_system=='AIX'
 
+    #IBM i requires fixes contained in numpy 1.23.3.
+    numpy==1.23.3; python_version=='3.9' and platform_system=='OS400'
```

Comment:
Why only for Python 3.9? The build issue seemed independent of Python version to me.

Reply:
Yes, we really have other python on i . Python 3.6 , but it's replaced by python 3.9 and all packages are based on python 3.9 only  now. So, it is ok we just support python 3.9 here.

Reply:
There's no way for me to check this, but - in numpy you used `os400` (lowercase), so are you sure it is `OS400` (uppercase) here? It's a string comparison, so case matters.